snapshot/btrfs/package.go

```diff
@@ -0,0 +1,7 @@
+package btrfs
```

Comment:
Let's try to avoid importing on those platforms. We don't want the package to build if it is not useful. ;)